libsolidity/ast/AST.cpp

```diff
@@ -290,6 +290,12 @@ TypeDeclarationAnnotation& EnumDefinition::annotation() const
 	return dynamic_cast<TypeDeclarationAnnotation&>(*m_annotation);
 }
 
+ContractDefinition::ContractKind FunctionDefinition::inContractKind() const
+{
+	return dynamic_cast<ContractDefinition const*>(scope()) &&
```

Comment:
I think this should assert if the scope is not set.

Reply:
True, this was a bad copy paste. This shouldn't even compile :)